package.json

```diff
@@ -37,10 +38,13 @@
   },
   "homepage": "https://github.com/asyncapi/spec-json-schemas#readme",
   "devDependencies": {
+    "eslint": "^8.54.0",
     "mocha": "^10.0.0",
     "nyc": "^15.1.0"
   },
   "dependencies": {
-    "@types/json-schema": "^7.0.11"
+    "@types/json-schema": "^7.0.11",
+    "ajv": "^8.12.0",
```

Comment:
Actually, those ajv dependencies could go un der devDependencies as you won't need it in the final npm package but only in development envs.

Reply:
Understood.
resolved with da96b36